spyder/locale/es/LC_MESSAGES/spyder.po

```diff
@@ -1422,19 +1427,21 @@ msgstr "Seleccionar una nueva fuente"
 
 #: spyder/plugins/completion/languageserver/confpage.py:179
 msgid "TCP port number of the server"
-msgstr ""
+msgstr "Número de puerto TCP del servidor"
 
 #: spyder/plugins/completion/languageserver/confpage.py:189
 msgid "Additional arguments required to start the server"
-msgstr ""
+msgstr "Argumentos adicionales son necesarios para iniciar el servidor"
```

Comment:
I think this one should be "Argumentos adicionales necesarios para iniciar el servidor"

Reply:
Right thanks! I'll fix it in the next commit.